Libplanet.Common/Libplanet.Common.csproj

```diff
@@ -1,32 +1,27 @@
 <Project Sdk="Microsoft.NET.Sdk">
-    <PropertyGroup>
-        <PackageId>Libplanet.Common</PackageId>
-        <Authors>Planetarium</Authors>
-        <Company>Planetarium</Company>
-        <PackageLicenseExpression>LGPL-2.1-or-later</PackageLicenseExpression>
-        <RequireLicenseAcceptance>true</RequireLicenseAcceptance>
-        <RepositoryUrl>https://github.com/planetarium/libplanet.git</RepositoryUrl>
-    </PropertyGroup>
-    
-    <PropertyGroup>
-        <LangVersion>8.0</LangVersion>
-        <Nullable>enable</Nullable>
-        <TargetFrameworks>netstandard2.1;netcoreapp3.1;netstandard2.0</TargetFrameworks>
-        <RootNamespace>Libplanet.Common</RootNamespace>
-        <IsTestProject>false</IsTestProject>
-        <CodeAnalysisRuleSet>..\Libplanet.ruleset</CodeAnalysisRuleSet>
-    </PropertyGroup>
+  <PropertyGroup>
+    <PackageId>Libplanet.Common</PackageId>
+    <Title>Libplanet.Common</Title>
+    <Authors>Planetarium</Authors>
+    <Company>Planetarium</Company>
+    <PackageLicenseExpression>LGPL-2.1-or-later</PackageLicenseExpression>
+    <RequireLicenseAcceptance>true</RequireLicenseAcceptance>
+    <RepositoryUrl>https://github.com/planetarium/libplanet.git</RepositoryUrl>
+  </PropertyGroup>
 
-    <ItemGroup>
-      <PackageReference Include="Bencodex" Version="0.10.0" />
-      <PackageReference Include="Bencodex.Json" Version="0.10.0" />
-      <PackageReference Include="BouncyCastle.Cryptography" Version="2.0.0" />
-      <PackageReference Include="Microsoft.Bcl.HashCode" Version="1.1.1" />
-      <PackageReference Include="System.Collections.Immutable" Version="1.7.1" />
-      <PackageReference Include="System.Diagnostics.Contracts" Version="4.3.0" />
-      <PackageReference Include="System.IO" Version="4.3.0" />
-      <PackageReference Include="System.Linq" Version="4.3.0" />
-      <PackageReference Include="System.Text.Json" Version="6.0.7" />
-    </ItemGroup>
```

Comment:
Is it intended?

Reply:
Yes? Partially moved down to a different place. Also, unless there is a good reason to use a specific `System` package versions, I don't think we should include them in the list.

Reply:
But if remove those packages, cannot build `Libplanet.Common`. Have you tried rebuilding with this PR?

Reply:
My mistake. 😓 Build well.